### PR TITLE
test: Fix English leak test for rate limit error

### PR DESCRIPTION
**Pull request by worldleelavivat-okta**
## Description:

Fix English leak test for rate limit error

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [x] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Reviewers:
- @haishengwu-okta 
- @nikhilvenkatraman-okta 
- @anipendakur-okta 

### Issue:

- [OKTA-390215](https://oktainc.atlassian.net/browse/OKTA-390215)

### CODE REVIEW

**Review by haishengwu-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -983,3 +983,6 @@ oie.browser.error.NotAllowedError = The operation either timed out or was not al
 # soft_token API errors
 authfactor.challenge.soft_token.invalid_passcode = Your code doesn't match our records. Please try again.
 authfactor.challenge.soft_token.used_passcode = Each code can only be used once. Please wait for a new code and try again.
+
+# rate limit
+tooManyRequests = Too many requests
```
Comment (nikhilvenkatraman-okta) [inline packages/@okta/i18n/src/properties/login.properties:988]: Can we add an `oie/idx` prefix to this key ?. It is a format that we follow for all oie keys. We are also looking to add some automation for customization where we need to know if a key is an oie key or not.

Comment (worldleelavivat-okta) [reply packages/@okta/i18n/src/properties/login.properties]: I double checked the error and look like `tooManyRequests` in the currently format is mainly used for OIE. 

I have added `oie` prefix to this key. The V1 error look more like error in `error-429-api-limit-exceeded.json`.

File: src/v2/ion/i18nTransformer.js [hunk 2]
```diff
@@ -128,6 +128,8 @@ const I18N_OVERRIDE_MAPPINGS = {
   'oie.session.expired' : 'oie.idx.session.expired',
 
   'activate-device.userCode' : 'oie.device.code.activate.label',
+
+  'tooManyRequests' : 'oie.tooManyRequests',
```
Comment (nikhilvenkatraman-okta) [inline src/v2/ion/i18nTransformer.js:132]: I don't think you need this change. This is only for input labels that are sent from the backend.

**Review by nikhilvenkatraman-okta — APPROVED** [review]
I think we can remove the i18nTransformer change.

**Review by nikhilvenkatraman-okta — CHANGES_REQUESTED** [review]
Looks like we dont have this change in the backend so this will break the widget. We should change the backend or revert the change for oie prefix

**Comment by worldleelavivat-okta** [comment]
@nikhilvenkatraman-okta reverted the prefix change

**Review by anipendakur-okta — APPROVED** [review]
lgtm

**Review by nikhilvenkatraman-okta — APPROVED** [review]

**Pull request closed without merging**